src/components/CreatePostInput/index.js

```diff
@@ -1,4 +1,5 @@
 import * as React from 'react';
+import PropTypes from 'prop-types';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import of `PropTypes` is a good addition for type checking in React. However, it's important to note that PropTypes only checks types in development mode and not in production. If you need type checking in production, consider using TypeScript.
```diff
+ import PropTypes from 'prop-types';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->